worlds/legacy_world_features.py

```diff
@@ -0,0 +1,34 @@
+# This file triggers various "legacy modes" for worlds, so that breaking changes
+# can be made without needing to simultaneously update or break every exisitng
+# world. Authors of worlds listed here should remove their worlds when they have
+# time to update them to the new behavior.
+
+from Options import PerGameCommonOptions
+from worlds.AutoWorld import AutoWorldRegister
+
+# # Option docstring format
+#
+# These worlds still use the old-style plain text Option docstrings. All other
+# worlds' Option docstrings are parsed as reStructuredText (the standard Python
+# docstring format) and rendered as HTML in the WebHost.
+for world_name in [
```

Comment:
This breaks APWorlds. Please don't do this. Let the world opt in.

Reply:
Imo breaking something purely visual for an apworld during AP's 0.# days, especially when apworld don't usually worry about being displayed on WebHost (cause how would they), needs to be okay

Reply:
However, with Phar's alternate suggestion of how to implement this, I'm now on board with this being opt in. Just saying right now that I disagree with your philisophy in general and it will probably come up again in the future

Reply:
Considering the long-term goal is decoupling worlds from core, I'm against adding anything that adds special privileges to worlds currently in the core repo (it also can't be edited easily from outside the apworld, if one of these world authors releases an updated apworld that would get placed in a webhost).

So regardless of what is decided (breaking existing APworlds or not), I am firmly against this file existing.

Reply:
To give some background here, this was intended as a combined solution for two problems:

1. This change shouldn't break the rendering for existing core worlds.
2. It's infeasible to update existing core worlds in-place because doing so would require a review from every world maintainer.

APWorlds could still easily keep the old behavior by adding `plain_text_doc = True` to their options (or `plain_text_options_doc = True` to their `World`). The idea isn't so much to give special privileges to core worlds as to work around how difficult it would be to treat them the same as APWorlds.

All that said, it sounds like the consensus is that the core maintainers prefer this to be opt-out. I worry that's not the right call in the long-term when plain text option docs provide no benefits other than backwards-compatibility, but I'll bow to your decision and make the change.